scripts/zkai_d128_value_adapter_policy_frontier_gate.py

```
@@ -0,0 +1,639 @@
+
+MUTATION_BUILDERS: tuple[tuple[str, MutationFn, bool], ...] = (
+    ("decision_promoted", lambda p: p.__setitem__("decision", "GO_VALUE_DERIVED_ADAPTER"), True),
+    ("claim_boundary_overclaim", lambda p: p.__setitem__("claim_boundary", "VALUE_DERIVED_ADAPTER_EXISTS"), True),
+    (
+        "best_admissible_mismatch_zeroed",
+        lambda p: p["summary"].__setitem__("best_admissible_mismatches", 0),
+        True,
+    ),
+    (
+        "index_only_admitted",
+        lambda p: p["policy_frontier"]["policies"][4].__setitem__("admissible_as_value_adapter", True),
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)